pymc/data.py

```diff
@@ -482,6 +482,8 @@ class Data:
     export_index_as_coords: bool, optional, default=False
```

Comment:
fix the link right above, it should not be html but `` :ref:`arviz:quickstart` ``

Reply:
Addressed via 56c45fa